toothpick-runtime/src/main/java/toothpick/configuration/Configuration.java

```diff
@@ -30,6 +30,7 @@
   private RuntimeCheckConfiguration runtimeCheckConfiguration = new RuntimeCheckOffConfiguration();
   private MultipleRootScopeCheckConfiguration multipleRootScopeCheckConfiguration =
       new MultipleRootScopeCheckOffConfiguration();
+  boolean usesScopes = true;
```

Comment:
This should be a configuration, similar to the multiple root config.

Reply:
changed